docusaurus/docs/how-to-guides/app-plugins/include-dashboards-in-app-plugins.md

```diff
@@ -93,12 +80,6 @@ To facilitate user customization, create a dashboard data source variable. This
 
 To test your newly created dashboard, import the dashboard:
 
-### Import a dashboard into a data source plugin
```

Comment:
@jackw does it make sense to keep Step 3 in its current state given app plugins automatically import?

The two suggestions I have are to change to "Step 3: Verify your dashboard imports successfully" or "Step 3: Add navigation links to your dashboard (optional)"

Reply:
```
## Step 3: Import the dashboard into your plugin

### Import a dashboard into an app plugin

### (Optional) Add navigation links in an app plugin

```
I don't think it makes sense if this doc is now app specific. I would opt for "Step 3: Add navigation links to your dashboard (optional)" and probably add something above in step 2 that explains they're auto imported... e.g.

```
4. After adding the dashboard to your plugin, rebuild the plugin and restart Grafana to apply the new configuration.
5. App dashboards are automatically imported by the Grafana server. Navigate to your dashboard to verify it works as intended.
```
Does that make sense? WDYT?

Reply:
made these changes in https://github.com/grafana/plugin-tools/pull/978/commits/772378ccc59cac9a65ce16511cef3efff287f26b minor rewording and shifted (optional) to end of line as that made it fit on a single line and looked better.